docs/3.x/tech/apis/authentication.adoc

```diff
@@ -6,20 +6,19 @@ description: Learn how the CleanSpeak APIs authenticate requests
 
 == Authentication
 
-CleanSpeak controls access to the API through the use of an API key. By default all APIs will return an `401 Unauthorized` response.
+CleanSpeak controls access to the API through the use of an API Key. By default all APIs will return an 401 Unauthorized response.
 
-To enable access to the API create one or more API keys in the CleanSpeak management interface. The API key is then supplied in the HTTP request using the Authorization header.  See API Keys for more information on adding additional keys.
+To enable access to the API create one or more API Keys in the CleanSpeak management interface. The API Key is then supplied in the HTTP request using the Authorization header.
```

Comment:
Some places you are using back ticks for `Authorization`, we should be consistent.